src/libraries/System.IO.Pipes/src/System/IO/Pipes/NamedPipeServerStream.cs

```diff
@@ -60,10 +59,10 @@ public NamedPipeServerStream(string pipeName, PipeDirection direction, int maxNu
         /// Win32 note: this gets used for dwPipeMode. CreateNamedPipe allows you to specify PIPE_TYPE_BYTE/MESSAGE
         /// and PIPE_READMODE_BYTE/MESSAGE independently, but this sets type and readmode to match.
         /// </param>
-        /// <param name="options">PipeOption enum: None, Asynchronous, or Write-through
+        /// <param name="options">PipeOption enum: None, Asynchronous, Write-through, or FirstPipeInstance
         /// Win32 note: this gets passed in with dwOpenMode to CreateNamedPipe. Asynchronous corresponds to
-        /// FILE_FLAG_OVERLAPPED option. PipeOptions enum doesn't expose FIRST_PIPE_INSTANCE option because
-        /// this sets that automatically based on the number of instances specified.
+        /// FILE_FLAG_OVERLAPPED option. PipeOptions.FIRST_PIPE_INSTANCE
+        /// is automatically set based on the number of instances specified.
```

Comment:
I don't understand what this comment is now referring to.  What is automatically setting FirstPipeInstance?